wandb/wandb_torch.py

```diff
@@ -42,6 +42,27 @@ def nested_shape(array_or_tuple):
         return []
 
 
+LOG_TRACK_COUNT, LOG_TRACK_THRESHOLD = range(2)
+
+
+def log_track_init(log_freq):
+    """create tracking structure used by log_track_update
+    """
+    l = [0] * 2
+    l[LOG_TRACK_THRESHOLD] = log_freq
+    return l
+
+
+def log_track_update(log_track):
+    """count (log_track[0]) up to threshold (log_track[1]), reset count (log_track[0]) and return true when reached
+    """
+    log_track[LOG_TRACK_COUNT] += 1
+    if log_track[LOG_TRACK_COUNT] <  log_track[LOG_TRACK_THRESHOLD]:
```

Comment:
Extra space here, I think I have a python formatter enabled we should probably standardize.